python/ometiff_tiler.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python3
+import pyvips
+import argparse
+import os
+
+# pyvips can run into issues with conda environments. Recommend to use a
+# clean environment, brew install vips and pip install pyvips
+
+
+def tile_ometiff(filename, channel_pages, output_directory):
+    for (channel, page) in channel_pages:
+        image = pyvips.Image.tiffload(filename, page=page)
+
+        path = os.path.join(
+            output_directory,
+            'linnarsson.images.{}'.format(channel)
```

Comment:
`linnarsson` should be passed in, if it's needed? Otherwise, it looks like this script is general purpose?